### DISCUSSION TITLE
kaslurp - a REPL wrapper

### DISCUSSION THREAD

**Discussion by jaygooby** [davidchalifoux/kaput-cli#21] [Show and tell]
👋 I made a small REPL wrapper for kaput: https://github.com/jaygooby/kaslurp

More details here: https://jay.gooby.org/2021/12/08/a-repl-wrapper-for-kaput-cli-a-put-io-client

**Comment by davidchalifoux** [comment]
Awesome work!

Thanks for the kind words, the criticism is appreciated as well. I'd love to implement some of the things you mentioned (especially multi-download) within Kaput, however I've since changed my media setup and no longer use Put. It's hard to justify spending time on something I'm not using and have to pay for 😅.

PRs are welcome should you want to contribute to Kaput directly!
